sos/plugins/manila.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2016 Red Hat, Inc., Vikrant Aggarwal <ervikrant06@gmail.com>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class ManilaPlugin(Plugin, RedHatPlugin):
+    """Openstack Manila"""
+    plugin_name = "manila"
+    profiles = ('openstack', 'openstack_controller', 'openstack_manila')
```

Comment:
Why define a profile for `openstack_manila`? If it is only one plugin then users can just as easily do `-o openstack_manila` (the whole point of profiles is that they are groups of related plugins).